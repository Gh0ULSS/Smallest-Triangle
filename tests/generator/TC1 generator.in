-N=3 -mindist=2 -rseed=15