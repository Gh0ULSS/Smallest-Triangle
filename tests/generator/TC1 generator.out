46.52, -48.83
23.6, -34.2
48.63, -48.31
